apollo-client/src/test/java/com/ctrip/framework/apollo/spring/JavaConfigAnnotationTest.java

```diff
@@ -91,6 +105,44 @@ public void testApolloConfigWithInheritance() throws Exception {
     assertEquals(applicationConfig, bean.getSomeConfig());
   }
 
+
+  @Test
+  public void testEnableApolloConfigResolveExpressionSimple() {
+    mockConfig(ConfigConsts.NAMESPACE_APPLICATION, mock(Config.class));
+    mockConfig("xxx", mock(Config.class));
+    getSimpleBean(TestEnableApolloConfigResolveExpressionWithDefaultValueConfiguration.class);
+  }
+
+  @Test
+  public void testEnableApolloConfigResolveExpressionFromSystemProperty() {
+    mockConfig(ConfigConsts.NAMESPACE_APPLICATION, mock(Config.class));
+
+    final String resolvedNamespaceName = "yyy";
+    System.setProperty("from.system.property", resolvedNamespaceName);
```

Comment:
It's better to clear the system property after the test.

Reply:
Found a concurrent problem.

Test 1 and test 2 use same key `simple.namespace` to set in their system property.

When test 1 running, test 2 finished, test 2 will clear `simple.namespace` which test 1 need, then test 1 failure.